### PR TITLE
Q1-2024 Sven Template Updates.md

### PR DESCRIPTION
**Pull request by sven-pa**

### CODE REVIEW

File: templates/general/checklist-mdr.md [hunk 1]
```diff
@@ -148,7 +148,7 @@
...
-| 17.3 | Software referred to in this Section that is intended to be used in combination with mobile computing platforms shall be designed and manufactured taking into account the specific features of the mobile platform (e.g. size and contrast ratio of the screen) and the external factors related to their use (varying environment as regards level of light or noise). | Yes |  | IEC 62304:2006 / AMD1:2015 | SOP Integrated Software Development<br>Software Requirements List |
+| 17.3 | Software referred to in this Section that is intended to be used in combination with mobile computing platforms shall be designed and manufactured taking into account the specific features of the mobile platform (e.g. size and contrast ratio of the screen) and the external factors related to their use (varying environment as regards level of light or noise). | No | Only applicable for mobile applications |  |  |
```
Comment (ivoflipse) [inline templates/general/checklist-mdr.md:151]: Medical Imaging products may have requirements for the environment or the screens they're viewed on. The FDA for example made quite a fuss about this for ONE Planner Hip

Comment (sven-pa) [reply templates/general/checklist-mdr.md:151]: Thanks for the pointer Ivo! But as this template refers to the MDR and we're trying to generally keep things minimalistic, I'd keep the wording as such for now.

File: templates/techdoc/62304/software-development-maintenance-plan.md [hunk 2]
```diff
@@ -52,8 +52,16 @@ Please see the relevant processes for the following activities:
 | Backend Developer  | 1     | Implementing Backend Software Requirements |
 
 ### 2.2 Software
-
-Describe your device's software safety class according to IEC 62304 and your reasoning behind the classification.
+**IEC 62304 Software Safety Classification**
+The software safety classification for [enter device name] has been established as class [XXXX] per IEC 62304:2006/AMD1:2015 based on the decision-making process outlined in table 3 and in paragraph 4.3 of the norm. A malfunction of, or latent design flaw in the software device may lead to situations with unacceptable risks [for example: false-positive and false-negative diagnosis, resulting in unnecessary interventions or missed necessary interventions]. This excludes software safety class A. Serious injuries or death, however, can be ruled out because [XXXX]. Considering these risk control measures external to the software system, safety class C can be ruled out, resulting in class B.
```
Comment (olieidel) [inline templates/techdoc/62304/software-development-maintenance-plan.md]: - Add a backslash `\` before bracket characters `[` `]`.
- Markdown paragraphs need an additional newline (enter) to be displayed as paragraph. Currently, the paragraphs here are only separated by one newline. Here's how it currently looks. You can check this yourself by viewing the markdown file in GitHub (if that makes sense).

Screenshot 2024-06-01 at 13 34 18

Comment (sven-pa) [reply templates/techdoc/62304/software-development-maintenance-plan.md]: Thanks - should be updated now!

**Review by olieidel — APPROVED** [review]

**Pull request merged**